supervisor/cmd/main.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
Mmmh... why isn't this package `cmd/supervisor`?

Reply:
to move it into the root .cosmos-sdk/cmd/supervisor? I was following more or less the simapp that lives in simapp/simd/cmd.

Reply:
I think this is OK and how I would do it

Reply:
Yeah, during our call today I asked Jonathan to ignore this comment of mine. Sorry for the noise!